internal/npm_install/test/generate_build_file.spec.js

```diff
@@ -94,4 +94,17 @@ describe('build file generator', () => {
       expect(bzl).toContain('def http_server(');
     });
   });
+
+  describe('getDirectDependencySet', () => {
+    it('returns a set of all dependencies in a package.json file', () => {
+      const pkgPath = path.join(
```

Comment:
@alexeagle What's the right way for this to work on windows? `require` the runfiles helper?

Reply:
Yup, get the runfiles library from env variable and do runfiles.require*

Reply:
example : https://github.com/bazelbuild/rules_nodejs/blob/065922b6963f1f9401c8a7638ef11c0709635886/internal/node/test/env.spec.js#L54